www-apps/otrs/otrs-5.0.15.ebuild

```diff
@@ -65,7 +65,7 @@ src_prepare() {
 	rm -fr "${S}/scripts"/{auto_*,redhat*,suse*,*.spec} || die
 	cp Kernel/Config.pm{.dist,} || die
 
-	# Fix broken png files
+	# Fix broken png file
 	pngfix -q --out=out.png "${S}/var/httpd/htdocs/skins/Agent/default/img/otrs-verify.png"
```

Comment:
OK, I got your comment that *pngfix* will oft end with non-zero exit code. But according to help non-zero exit code >=16 indicates an uncorrectable error. Failures 1-15 are just a notice about an correctable error. So let's add

```bash
if [[ $? -gt 15 ]]; then
	die "pngfix failed"
fi
```

and update comment to include "(and see pngfix help for exit codes)"